plugins/source/gcp/codegen/storage.go

```diff
@@ -14,13 +12,6 @@ var storageResources = []*Resource{
 		ResponseStruct: &storage.BucketAttrs{},
 		SkipFetch:      true,
 		SkipMock:       true,
-		OverrideColumns: []codegen.ColumnDefinition{
```

Comment:
`BucketAttrs` doesn't have a self link field so adding `OverrideColumns` didn't have an impact of the generated code